include/pybind11/detail/common.h

```diff
@@ -369,6 +396,23 @@ constexpr size_t instance_simple_holder_in_ptrs() {
     return size_in_ptrs(sizeof(std::shared_ptr<int>));
 }
 
+inline bool Py_VersionCheckPassed() {
```

Comment:
Similar to my other comment, we should consistently replace the `PYBIND11_CHECK_PYTHON_VERSION` macro.

Reply:
Done. Thanks!